src/ddmd/mars.d

```diff
@@ -245,6 +245,29 @@ extern (C++) void genCmain(Scope* sc)
     rootHasMain = sc._module;
 }
 
+private void semantic3OnDependencies(Module m, StringTable *importArray)
+{
+    if(!m)
+        return;
+
+    const(char) *mName = m.toChars();
+    size_t mNameLen    = strlen(mName);
+
+    if (importArray.lookup(mName, mNameLen))
+        return;
+
+    if (global.params.verbose)
+        fprintf(global.stdmsg, "semantic3 %s\n", mName);
+
+    importArray.insert(mName, mNameLen, cast(void*) mName);
```

Comment:
Yah, the styles are quite different so that would be a large project. I think it would be a palpable possibility once we have a complete pretty printer with LaTeX's algorithm for line breaking.